src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerworker.cpp

```diff
@@ -36,8 +36,24 @@ void PlatformInconsistencyCheckerWorker::execute() {
 
     _idsToBeRemoved.clear();
 
-    ExitCode exitCode = checkTree(_syncPal->updateTree(ReplicaSide::Remote)->rootNode(),
-                                  _syncPal->updateTree(ReplicaSide::Remote)->rootNode()->name());
+    ExitCode exitCodeRemote = checkRemoteTree(_syncPal->updateTree(ReplicaSide::Remote)->rootNode(),
+                                              _syncPal->updateTree(ReplicaSide::Remote)->rootNode()->name());
+
+    ExitCode exitCodeLocal = checkLocalTree(_syncPal->updateTree(ReplicaSide::Local)->rootNode(),
+                                            _syncPal->updateTree(ReplicaSide::Local)->rootNode()->name());
+
+    ExitCode exitCode = ExitCode::Ok;
```

Comment:
**Suggestion.** Make this logging happen within `checkTree(const ReplicaSide side)`. This way you will reduce redundancy.